<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<title>Articles</title>
		<link href="http://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet" type="text/css" />
		<link href="http://fonts.googleapis.com/css?family=Kreon" rel="stylesheet" type="text/css" />
		<link rel="stylesheet" type="text/css" href="formstyle.css" />
	</head>
	<body>
		<div id="wrapper">
			<div id="header">
				<div id="logo">
					<h1><a href="#">Covi-Care Blogs</a></h1>
				</div>
				<div id="menu">
					<ul>
						<li class="first active"><a href="#">Home</a></li>
						<li><a href="#">Blog2</a></li>
						<li><a href="#">Blog3</a></li>
                        <li><a href="#">Blog3</a></li>
                        <li><a href="#">Blog4</a></li>
						
					</ul>
					<br class="clearfix" />
				</div>
			</div>
			<div id="page">
				<div id="sidebar">
					<div class="box">
						<h3>Blogs List</h3>
						
						<ul class="list">
							<li class="first"><a href="#">THE RISE AND IMPACT OF COVID-19 IN INDIA</a></li>
							<li><a href="#">The proximal origin of SARS-CoV-2</a></li>
							<li class="last"><a href="#">Covid-19 — Navigating the Uncharted</a></li>
                            <li class="last"><a href="#">Treatment of 5 Critically Ill Patients With COVID-19 With Convalescent Plasma</a></li>
                            <li class="last"><a href="#">Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China</a></li>
						</ul>
					</div>
					<div class="box">
						
					</div>
				</div>
				<div id="content">
					<div class="box">
						<h2>The proximal origin of SARS-CoV-2</h2>
						<img class="alignleft" src="blogimg/pic02.jpg" width="200" height="180" alt="" />
						<p>
						
						</p>
					</div>
					<div class="box">
						
						<p>
							SARS-CoV-2 is the seventh coronavirus known to infect humans; SARS-CoV, MERS-CoV and SARS-CoV-2 can cause severe disease, whereas HKU1, NL63, OC43 and 229E are associated with mild symptoms6. Here we review what can be deduced about the origin of SARS-CoV-2 from comparative analysis of genomic data. We offer a perspective on the notable features of the SARS-CoV-2 genome and discuss scenarios by which they could have arisen. Our analyses clearly show that SARS-CoV-2 is not a laboratory construct or a purposefully manipulated virus.
						</p>
					</div>
					<div class="box">
						
						<p>
							Our comparison of alpha- and betacoronaviruses identifies two notable genomic features of SARS-CoV-2: (i) on the basis of structural studies7,8,9 and biochemical experiments1,9,10, SARS-CoV-2 appears to be optimized for binding to the human receptor ACE2; and (ii) the spike protein of SARS-CoV-2 has a functional polybasic (furin) cleavage site at the S1–S2 boundary through the insertion of 12 nucleotides8, which additionally led to the predicted acquisition of three O-linked glycans around the site.</p><p>

1. Mutations in the receptor-binding domain of SARS-CoV-2
The receptor-binding domain (RBD) in the spike protein is the most variable part of the coronavirus genome1,2. Six RBD amino acids have been shown to be critical for binding to ACE2 receptors and for determining the host range of SARS-CoV-like viruses7. With coordinates based on SARS-CoV, they are Y442, L472, N479, D480, T487 and Y4911, which correspond to L455, F486, Q493, S494, N501 and Y505 in SARS-CoV-27. Five of these six residues differ between SARS-CoV-2 and SARS-CoV (Fig. 1a). On the basis of structural studies7,8,9 and biochemical experiments1,9,10, SARS-CoV-2 seems to have an RBD that binds with high affinity to ACE2 from humans, ferrets, cats and other species with high receptor homology7.
						</p><p>
                        While the analyses above suggest that SARS-CoV-2 may bind human ACE2 with high affinity, computational analyses predict that the interaction is not ideal7 and that the RBD sequence is different from those shown in SARS-CoV to be optimal for receptor binding7,11. Thus, the high-affinity binding of the SARS-CoV-2 spike protein to human ACE2 is most likely the result of natural selection on a human or human-like ACE2 that permits another optimal binding solution to arise. This is strong evidence that SARS-CoV-2 is not the product of purposeful manipulation.</p>
					</div>
					<br class="clearfix" />
				</div>
				<br class="clearfix" />
			</div>
			
		</div>
		
	</body>
</html>